EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78074
+stocksregister.com#?#div[id^="stock-"]:has(> p[style] > a[href*="/aff_c?offer_id="])
```

Comment:
I'd use `stocksregister.com#?#.td-post-content > div[id^="stock-"]:has(> p[style] > span[style]:contains(Sponsored))`.

Reply:
I'd use `stocksregister.com#?#div[id^="stock-"]:has(> p[style] > a[href*="/aff_c?offer_id="])`. It's narrowed enough already.

Reply:
It `a[href*="/aff_c?offer_id="]` would be great if we didn't have that `Sponsored`.

Reply:
C'mon it's the same thing.

Reply:
Not really. I've already explained the difference.